packages/carousel/src/constants.js

```diff
@@ -1,17 +1,17 @@
-const XS_VISIBLE_SLIDES = 1.1
+const VISIBLE_SLIDES_BREAKPOINT_1 = 1.1
 
-const SM_VISIBLE_SLIDES = 2.1
+const VISIBLE_SLIDES_BREAKPOINT_2 = 2.1
```

Comment:
what's the rational behind these names vs the previous ones? matching the named breakpoints makes more sense than numbers imo

Reply:
Renamed because they don't actually map to the regular DS breakpoints.

Reply:
that's fair lol is one the smallest and the other the biggest breakpoint that they render for?

Reply:
Yeah, the carousel only has 2 breakpoints for customizing currently.

Reply:
would `VISIBLE_SLIDES_BREAKPOINT_SMALLEST` and `VISIBLE_SLIDES_BREAKPOINT_LARGEST` fit the situation better? I'm not 100% on what these variables are used for

Reply:
Possibly, if we end up adding another one though, i think it would be hard to name `VISIBLE_SLOIDES_BREAKPOINT_MEDIUMIST`